exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -116,11 +117,19 @@ def invalid_url?(url)
           true
         end
 
+        def send_bytes_hook
+          if !@send_bytes_hook.nil?
+            @send_bytes_hook.call { yield }
+          else
+            OpenTelemetry::Common::Utilities.untraced { yield }
+          end
+        end
```

Comment:
I'm not sure whether this explicit hook argument is better than
```ruby
def trace_request
  OpenTelemetry::Common::Utilities.untraced { yield }
end
```
and then relying on subclassing or `prepend` to override the behaviour.

Reply:
Or even just a boolean flag `should_trace_request: false` and then:
```ruby
def trace_request
  if @should_trace_request
    tracer.in_span('OTLP::Exporter.send_bytes') { yield }
  else
    OpenTelemetry::Common::Utilities.untraced { yield }
  end
end
```

Reply:
The first option is more flexible, since it allows adding attributes.

Reply:
I'll take the first suggested approach here.